src/test/java/aquarium/shop/ProductControllerMockTest.java

```diff
@@ -29,16 +40,24 @@
     @Autowired
     private MockMvc mvc;
 
-    @Test
-    public void getAllProducts() throws Exception {
+    private static List<Product> products = new ArrayList<>();
+
+    @BeforeEach
```

Comment:
Please add also tearDown method which will remove product added here from list. In other case there will be unnecessary elements in products list in later tests.